com.rq3.Reaction.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: com.rq3.Reaction
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: Reaction
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --persist=.Reaction
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/sdl12-compat.json
+
+  - name: reaction
+    buildsystem: simple
+    build-commands:
+      - make DEFAULT_BASEDIR=${FLATPAK_DEST}/share/Reaction -j${FLATPAK_BUILDER_N_JOBS}
+      - mkdir -p ${FLATPAK_DEST}/bin/
+      - mkdir -p ${FLATPAK_DEST}/share/Reaction/
+      - make copyfiles COPYBINDIR=${FLATPAK_DEST}/bin/ COPYDIR=${FLATPAK_DEST}/share/Reaction/ NO_STRIP=1
+      - mv ${FLATPAK_DEST}/bin/Reactionded.* ${FLATPAK_DEST}/bin/Reactionded
+      - mv ${FLATPAK_DEST}/bin/Reaction.* ${FLATPAK_DEST}/bin/Reaction
+      - mv ${FLATPAK_DEST}/bin/*.so ${FLATPAK_DEST}/share/Reaction/
+      - install -Dm644 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 reaction.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
```

Comment:
why aren't these two name properly from the get go since you submitted the patch?

Reply:
I package software longer than Flathub exists and these naming rules are new and a bit counter-intuitive.